### PR TITLE
added BOLD_FRAME and INTERIOR_MEDIUM_FRAME

### PR DESCRIPTION
**Pull request by TaxiService**
added two new frame borders 
- `INTERIOR_MEDIUM_FRAME` is a version of `MEDIUM_FRAME` without the `DFHack` text at the bottom.
- `BOLD_FRAME` uses a new border texture (border-bold.png), and is meant to be used for important panels.

### CODE REVIEW

**Comment by myk002** [comment]
split from #3123 (commenting to tag the other PR with a reference to this one)

**Comment by TaxiService** [comment]
nice! i'll do it for the next branches as i make 'em

**Comment by myk002** [comment]
could you also add a note to the changelog (under "Lua", I think)

**Review by myk002 — COMMENTED** [review]
could you also add a note to the changelog (under "Lua", I think)?

**Comment by myk002** [comment]
build action failures appear to be a GitHub issue. They'll re-run (and hopefully pass -- they should) when you push the changelog commit.

**Comment by TaxiService** [comment]
oh my god--- i always forget that the changelog is included, and that _i can change it_ 0_0
yeah, sorry-- i'll also add changelog entires for all these prs!!

File: docs/changelog.txt [hunk 1]
```diff
@@ -48,6 +48,7 @@ changelog.txt uses a syntax similar to RST, with a few special sequences:
 ## API
 
 ## Lua
+- `widget`: added two new window borders, ``BOLD_FRAME`` and ``INTERIOR_MEDIUM_FRAME``
```
Comment (myk002) [inline docs/changelog.txt:51]:
```suggestion
- added two new window borders: ``gui.BOLD_FRAME`` for accented elements and ``gui.INTERIOR_MEDIUM_FRAME`` for a signature-less frame that's thicker than the existing ``gui.INTERIOR_FRAME``
```

**Pull request merged**